Data/Constants.cs

```diff
@@ -1,13 +1,45 @@
-﻿namespace DICUI.Data
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows.Media;
+
+namespace DICUI.Data
 {
     /// <summary>
-    /// Text for UI elements
+    /// Variables for UI elements
     /// </summary>
     public static class UIElements
```

Comment:
Nitpick: If there is more than just constant values in here now, I'd suggest moving UIElements to either its own file or to another, more matching one